.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,22 @@
+# https://pre-commit.com
+# This GitHub Action assumes that the repo contains a valid .pre-commit-config.yaml file.
+# Using pre-commit.ci is even better that using GitHub Actions for pre-commit.
+name: pre-commit
+on:
+  pull_request:
+    branches: [master]
+  push:
+#    branches: [master]
```

Comment:
I would typically remove dead code rather than leaving it commented since it's unclear to the reader why it was commented out or whether it's intended to be restored in the future.

Reply:
This is a tricky one...  If I don't comment this out then the tests do not run.  I will make a new PR to uncomment it because now that this has been landed, it will run.  Just a querk of GitHub Actions.